gh-144203: Customizable venv prompt prefix and suffix (#144205)
python/cpython: Misc/NEWS.d/next/Tools-Demos/2026-01-24-04-14-58.gh-issue-144203.PvBQZp.rst

> venv prompt in your shell can be customizable for prefix and suffix of venv name using VIRTUAL_ENV_PROMPT_PREFIX and VIRTUAL_ENV_PROMPT_SUFFIX from the default "(" and ") ".

Author: Please suggest if Tools-Demos is not correct.

Reviewer: Please use Library

> venv prompt in your shell can be customizable for prefix and suffix of venv name using VIRTUAL_ENV_PROMPT_PREFIX and VIRTUAL_ENV_PROMPT_SUFFIX from the default "(" and ") ".
> If variable is not set, the default "(" and ") " still be used.

Reviewer 2: Small wording suggestion:

```suggestion
The venv prompt in your shell can be customized for the prefix and suffix of the venv name using VIRTUAL_ENV_PROMPT_PREFIX and VIRTUAL_ENV_PROMPT_SUFFIX instead of  "(" and ") ".
If these variables are not set, the default "(" and ") " will be used.
```
